### PR TITLE
Add basic 2016 stats post

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/blog/write-the-docs-2016-year-in-review.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+Write the Docs 2016 year in review
+==================================
+
+2016 was a wonderful year for Write the Docs.
+This is our fourth year of existence,
+and this is the first year it feels that we have coalesced into a more cohesive community.
+This post will cover a few stats and other tidbits that we feel mark our progress,
+and we hope to continue to report these each year moving forward.
+
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2016-year-in-review.rst:11]: Maybe something like 

> 2016 was a wonderful year for Write the Docs, it marks our fourth year of existence,
> and was the year that we coalesced into a more cohesive community.

> This post covers a few stats and tidbits of information that we feel mark our progress, we'll be reporting these each year from here on in.

Comment (plaindocs) [reply docs/blog/write-the-docs-2016-year-in-review.rst:11]: I'm not majorly happy with my suggestions either, but they do read a little less clunky. Maybe @thatdocslady can help us out?

File: docs/blog/write-the-docs-2016-year-in-review.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Conferences
+===========
+
+* 400 attendees in Portland
+* 250 attendees in Prague
+
+The :doc:`conferences </conf/index>` continue to be the yearly physical manifestation of our community.
+They are the time each year where hundreds of wonderful folks gather and have a great time,
+while learning so much from each other.
+Proving each year that work can be both fun and informative is one of my greatest pleasures.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2016-year-in-review.rst:21]: > The :doc:`conferences </conf/index>` continue to be the yearly physical manifestation of our community in Europe and the USA. They are the time and place where hundreds of wonderful folks gather and have a great time, while learning so much from each other.

File: docs/blog/write-the-docs-2016-year-in-review.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+We continue to publish :doc:`videos </videos>` freely to the public.
+This is done so that the information can be most widely shared,
+and help people who may not be able to attend the events.
+Write the Docs always strives to be as inclusive as possible,
+and continuing to publish these videos is part of our commitment to openness.
+
+Slack
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2016-year-in-review.rst]: > Slack network? Slack chat? Slack community chat? I dunno

File: docs/blog/write-the-docs-2016-year-in-review.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+=====
+
+* 1,200 members
+
+This is the part of the community that I've been most happy with this year.
+Our :doc:`Slack network </slack>` has really caught on,
+and folks are discussing a wide range of topics every day.
+It really feels like this is where the community is thriving,
+and it becoming the default way of contacting folks inside the Write the Docs community.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2016-year-in-review.rst]: > and it is

File: docs/blog/write-the-docs-2016-year-in-review.rst [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+Newsletter
+==========
+
+* 2,485 subscribers
+
+The :doc:`newsletter </newsletter>` has been the second initiative that has really taken off this year.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2016-year-in-review.rst:45]: You're switching between this year / last year a bit I think.

> The :doc:`newsletter </newsletter>` is the second initiative that really took off in 2016.

File: docs/blog/write-the-docs-2016-year-in-review.rst [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+We had a few conversations about how to archive Slack in a good way,
+and the idea of sending summary updates each month was an inventive way to handle this.
+I think it has been full of value each month,
+and a wonderful way to keep tabs on the the community if you can't keep up with Slack.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2016-year-in-review.rst]: > and is a wonderful...

File: docs/blog/write-the-docs-2016-year-in-review.rst [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+Podcast
+=======
+
+* 2 episodes
+
+A few different folks have talked about doing a :doc:`podcast </podcast>`,
+and this year we finally got the initiative off the ground.
+This is a fully community run podcast,
+and was organized by members in the community.
+The first two episodes are off the ground,
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2016-year-in-review.rst]: Repeated off the ground.

File: docs/podcast.rst [hunk 8]
```diff
@@ -0,0 +1,14 @@
+Write the Docs Podcast
+======================
+
+.. note:: You can see more about our `podcast`_ on it's own dedicated site: http://podcast.writethedocs.org/
+
+Write the Docs is a community of people focused on software documentation. We consider everyone who cares about communication, documentation, and their users to be a member of our community. This can be programmers, tech writers, customer support, marketers, and anyone else who wants people to have great experiences with software.
```
Comment (plaindocs) [inline docs/podcast.rst:6]: developer advocates :-p

File: docs/blog/write-the-docs-2016-year-in-review.rst (hunk 5 above)
Comment (ericholscher) [reply docs/blog/write-the-docs-2016-year-in-review.rst:45]: Looks like I generally say "this year" meaning the year we're talking about. Seems fine.

File: docs/podcast.rst (hunk 8 above)
Comment (ericholscher) [reply docs/podcast.rst:6]: Heh, this is just copied from the podcast site, so just keeping it the same.

**Comment by plaindocs** [comment]
LGTM

**Pull request merged**